lib/foreman_maintain/package_manager/yum.rb

```diff
@@ -38,8 +38,12 @@ def installed?(packages)
       sys.execute?(%(rpm -q #{packages_list}))
     end
 
-    def find_installed_package(name)
-      status, result = sys.execute_with_status(%(rpm -q '#{name}'))
+    def find_installed_package(name, queryfm = '')
```

Comment:
I'd call this param `queryfmt` or `queryformat`, `fmt` is a common abbreviation for "format" and is also used in `man rpm` to describe the *value* passed after `--queryformat`. `fm` on the other hand can mean `foreman-maintain` and that initially confused me.

Reply:
Thanks for the review, as I am at changing param name, I changed it for list_installed_packages as well.